.travis.yml

```diff
@@ -1,24 +1,26 @@
 language: c
+git:
+  submodules: false
+
+env: B=autotools
+
 install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq curl
   - curl http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/xUbuntu_12.04/Release.key | sudo apt-key add -
```

Comment:
OK, but previously we had to install it and that time Travis was also a development CI and not a desktop system ;-)
( and travis.yml could document how to setup a  build environment to build syslog-ng from source even on a desktop system, but I'm ok with this change )